docs/resources/resource_provider_functions.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
I think we don't need this doc, because we have docs for each provider function.